chunky/src/java/se/llbit/chunky/main/Chunky.java

```diff
@@ -52,21 +62,12 @@
 import se.llbit.log.Receiver;
 import se.llbit.util.TaskTracker;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.nio.file.Path;
-import java.util.HashSet;
-import java.util.Set;
-
 /**
- * Chunky is a Minecraft mapping and rendering tool created by
- * Jesper Öqvist (jesper@llbit.se).
+ * Chunky is a Minecraft mapping and rendering tool created by Jesper Öqvist (jesper@llbit.se).
  *
  * <p>Read more about Chunky at <a href="https://chunky.llbit.se">https://chunky.llbit.se</a>.
  */
-public class Chunky {
+public class Chunky implements BlockProviderRegistry {
```

Comment:
It feels a bit strange that `Chunky` itself is the `Registry` but it shouldn't be a problem

Reply:
That's why I made it an interface. This allows us to let anything else implement the registry later without much changes